TransmissionProject/Transmission/2.94.yaml

```diff
@@ -0,0 +1,15 @@
+Id: TransmissionProject.Transmission
+Version: 2.94
+Name: Transmission
+Publisher: Transmission Project
+License: GPL
+LicenseUrl: https://github.com/transmission/transmission/blob/master/COPYING
+AppMoniker: transmission
+Tags: torrent, p2p, bittorrent
+Description: A Fast, Easy, and Free BitTorrent Client
+Homepage: https://transmissionbt.com
+Installers:
+  - Arch: x64
+    Url: https://github.com/transmission/transmission-releases/raw/master/transmission-2.94-x64.msi
+    Sha256: 
```

Comment:
The Sha256 hash for TransmissionQt is A35672252661F16382072FD0AE87B8EAFD47F5535439336FE2F161A542F9A1AB